tests/snippets/conftest.py

```diff
@@ -18,6 +18,7 @@
 
 import pathlib
 import pytest
+from pathlib import Path
```

Comment:
Same.

Reply:
I think this was a sloppy workaround for a change from the main repo that was once broken but has since resolved itself. This file should match now.